initrd/bin/seal-hotpkey

```diff
@@ -121,38 +130,50 @@ if [ "$((now_date - gpg_key_create_time))" -gt "$month_secs" ]; then
 elif [ "$admin_pin_retries" -lt 3 ]; then
```

Comment:
Something's not right here on Librem Key, I'm getting:

`/bin/seal-hotpkey: line 130: [: Admin3,User3: integer expression expected`

(it's getting `Admin3,User3` in admin_pin_retries)

I can help figure this out but it'll have to be in January

Reply:
fixed under https://github.com/linuxboot/heads/pull/1875/commits/696ecf54cd38d278495a3119ca8afcd627d6d2bb and tested with undusted old 0.10 fw based librem key I had in toolbox. <Nk3 works.